tests/e2e/utils/auto-rds-s3-setup.py

```diff
@@ -543,23 +496,42 @@ def setup_kubeflow_pipeline():
     print("Retrieving DB instance info...")
     try:
         db_instance_info = get_db_instance_info()
-    except get_rds_client().exceptions.DBInstanceNotFoundFault:
+    except get_rds_client(CLUSTER_REGION).exceptions.DBInstanceNotFoundFault:
         print("Could not retrieve DB instance info, aborting Kubeflow pipeline setup!")
         return
 
     pipeline_rds_params_env_file = "../../../awsconfigs/apps/pipeline/rds/params.env"
-    pipeline_rds_secret_provider_class_file = "../../../awsconfigs/common/aws-secrets-manager/rds/secret-provider.yaml"
-    pipeline_rds_params_env_lines = get_pipeline_params_env_lines(pipeline_rds_params_env_file)
-    new_pipeline_rds_params_env_lines = get_updated_pipeline_rds_params_env_lines(db_instance_info, pipeline_rds_params_env_lines)
-    edit_pipeline_params_env_file(new_pipeline_rds_params_env_lines, pipeline_rds_params_env_file)
-    update_secret_provider_class(pipeline_rds_secret_provider_class_file, RDS_SECRET_NAME)
+    pipeline_rds_secret_provider_class_file = (
+        "../../../awsconfigs/common/aws-secrets-manager/rds/secret-provider.yaml"
+    )
+    pipeline_rds_params_env_lines = get_pipeline_params_env_lines(
+        pipeline_rds_params_env_file
+    )
+    new_pipeline_rds_params_env_lines = get_updated_pipeline_rds_params_env_lines(
+        db_instance_info, pipeline_rds_params_env_lines
+    )
+    edit_pipeline_params_env_file(
+        new_pipeline_rds_params_env_lines, pipeline_rds_params_env_file
+    )
+    update_secret_provider_class(
+        pipeline_rds_secret_provider_class_file, RDS_SECRET_NAME
+    )
 
     pipeline_s3_params_env_file = "../../../awsconfigs/apps/pipeline/s3/params.env"
-    pipeline_s3_secret_provider_class_file = "../../../awsconfigs/common/aws-secrets-manager/s3/secret-provider.yaml"
-    pipeline_s3_params_env_lines = get_pipeline_params_env_lines(pipeline_s3_params_env_file)
-    new_pipeline_s3_params_env_lines = get_updated_pipeline_s3_params_env_lines(pipeline_s3_params_env_lines)
-    edit_pipeline_params_env_file(new_pipeline_s3_params_env_lines, pipeline_s3_params_env_file)
-    update_secret_provider_class(pipeline_s3_secret_provider_class_file, S3_SECRET_NAME)
+    pipeline_s3_secret_provider_class_file = (
+        "../../../awsconfigs/common/aws-secrets-manager/s3/secret-provider.yaml"
+    )
+    pipeline_s3_params_env_lines = get_pipeline_params_env_lines(
+        pipeline_s3_params_env_file
+    )
+    new_pipeline_s3_params_env_lines = get_updated_pipeline_s3_params_env_lines(
+        pipeline_s3_params_env_lines
+    )
+    edit_pipeline_params_env_file(
```

Comment:
I think we can reduce duplication here as well by using write_env_file or somthing we have in our utils but ok for now

Reply:
https://github.com/awslabs/kubeflow-manifests/blob/8c770276d98e3236d4f774e0cb459b68467656e1/tests/e2e/utils/config.py#L110